packages/react-components/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -184,258 +174,246 @@ const TicketOrderFormInputsAccessibility = () => {
+                    {('minLength' in errors.password.types || 'maxLength' in errors.password.types) && (
+                      <li>Have between 8 and 20 characters.</li>
+                    )}
+                    {('hasLowercaseLetter' in errors.password.types ||
+                      'hasUppercaseLetter' in errors.password.types ||
+                      'hasSpecialChar' in errors.password.types ||
+                      'hasNumber' in errors.password.types ||
+                      'noWhiteSpace' in errors.password.types) && (
+                      <li>
+                        Contain at least one lower case letter, upper case letter, number, special character and no
+                        spaces.
+                      </li>
+                    )}
+                  </ul>
+                </>
+              )}
+            </ValidationMessage>
+          )}
+
+          <Label htmlFor="birthDate">Birth date:</Label>
+          <Controller
+            name="birthDate"
+            control={control}
+            as={
+              <Input
+                type="text"
+                id="birthDate"
+                placeholder="E.g. 3/21/1995"
+                aria-required="true"
+                aria-invalid={!!errors.birthDate}
+                aria-describedby="birthDateErrors"
+              />
+            }
+            rules={{
+              required: true,
+              validate: {
+                validDate: value => regexes.validDate.test(value),
+                always: () => {
+                  if (!formState.isSubmitting) {
+                    formValidation.onFieldValidated('birthDate');
+                  }
+                  return true;
+                },
+              },
+            }}
+          />
+          {errors.birthDate?.types && (
+            <ValidationMessage id="birthDate" formValidation={formValidation}>
+              {'required' in errors.birthDate.types ? (
+                <p>Birth date is required.</p>
+              ) : (
+                <>
+                  <p>Birth date is invalid. It must:</p>
+                  <ul>{'validDate' in errors.birthDate.types && <li>Be in the MM/DD/YYYY format.</li>}</ul>
+                </>
+              )}
+            </ValidationMessage>
+          )}
+
+          <Checkbox label="Send me newsletter" onChange={onSendNewsletterChange} />
+
+          <Label htmlFor="email">E-mail:</Label>
+          <Controller
+            name="email"
+            control={control}
+            as={
+              <Input
+                type="text"
+                id="email"
+                disabled={!isSendNewsletter}
+                aria-required={isSendNewsletter}
+                aria-invalid={!!errors.email}
+                aria-describedby="emailErrors"
+              />
+            }
+            rules={{
+              required: isSendNewsletter,
+              validate: {
+                validEmail: value => regexes.validEmail.test(value),
```

Comment:
the validate function for Email should check for `isSendNewsletter`, since it's erroring now even if the send newsletter checkbox is unchecked.

Reply:
Thanks. I hav made the fix.